components/prism-dataweave.js

```diff
@@ -0,0 +1,48 @@
+(function (Prism) {
+    var keywords = /\b(?:match|input|output|ns|type|update|null|if|else|using|unless|at|is|as|case|do|fun|var|not|and|or)\b/;
+    
+    Prism.languages.dataweave = {	
+        "url": {
+            pattern:/(?:[A-z])+:\/\/(?:[A-z0-9/:\.\-_?=&])+|urn:(?:[A-z0-9:\-_\.?=&])+/
```

Comment:
(Please apply the fix for `A-z` here as well.)

```suggestion
            pattern:/\b[A-z]+:\/\/[A-z0-9/:\.\-_?=&]+|\burn:[A-z0-9:\-_\.?=&]+/
```